src/Components/MovieCard/MovieCard.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Movie } from '../../Models/Movie';
+import css from './MovieCard.less';
+
+export function MovieCard({ movie }: { movie: Movie; key: number; }) {
```

Comment:
You don't need to declare key here because keys don’t get passed to your components